lib/ansible/modules/network/infinity/infinity.py

```diff
@@ -0,0 +1,650 @@
+# !/usr/bin/python
+# meiliu@fusionlayer.com
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
```

Comment:
All modules must have the following sections defined in this order:

All modules must have the following sections defined in this order:

Copyright
ANSIBLE_METADATA
DOCUMENTATION
EXAMPLES
RETURNS
Python imports


See http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html